swagger.json

```diff
@@ -4822,6 +4904,156 @@
+            "properties": {
+                "timestampMs": {
+                    "type": "integer",
+                    "description": "Timestamp that the harsh event occurred in Unix milliseconds since epoch",
+                    "example": 1535590776000
+                },
+                "harshEventType": {
+                    "type": "string",
+                    "description": "Type of the harsh event",
+                    "example": "Harsh Braking"
+                }
+            }
+        },
+        "VehicleSafetyScoreResponse": {
+            "type": "object",
+            "description": "Safety score details for a vehicle",
+            "properties": {
+                "vehicleId": {
+                    "type": "integer",
+                    "description": "Vehicle ID",
+                    "example": 4321
+                },
+                "safetyScore": {
+                    "type": "integer",
+                    "description": "Safety Score",
+                    "example": 97
+                },
+                "safetyScoreRank": {
+                    "type": "string",
+                    "description": "Safety Score Rank",
+                    "example": "26"
+                },
+                "totalDistanceDrivenMeters": {
+                    "type": "integer",
+                    "description": "Total distance driven in meters",
+                    "example": 291836
+                },
+                "totalTimeDrivenMs": {
+                    "type": "integer",
+                    "description": "Amount of time driven in milliseconds",
+                    "example": 19708293
+                },
+                "timeOverSpeedLimitMs": {
+                    "type": "integer",
+                    "description": "Amount of time driven over the speed limit in milliseconds",
+                    "example": 3769
+                },
+                "totalHarshEventCount": {
+                    "type": "integer",
+                    "description": "Total harsh event count",
+                    "example": 3
+                },
+                "harshEvents": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/SafetyReportHarshEvent"
+                    }       
+                },
+                "harshBrakingCount": {
+                    "type": "integer",
+                    "description": "Harsh braking event count",
+                    "example": 2
+                },
+                "harshAccelCount": {
+                    "type": "integer",
+                    "description": "Harsh acceleration event count",
+                    "example": 1
+                },
+                "harshTurningCount": {
+                    "type": "integer",
+                    "description": "Harsh turning event count",
+                    "example": 0
+                },
+                "crashCount": {
```

Comment:
Does it make sense to share this model definition with `DriverSafetyScoreResponse`?

Reply:
so they are the exact same, except the `VehicleSafetyScoreResponse` has `vehicleId` instead of `driverId`. I wasn't sure if there was a way to consolidate all the other shared properties though...